package.json

```diff
@@ -224,6 +224,7 @@
   "dependencies": {
     "@babel/runtime": "^7.23.2",
     "@blockaid/ppom_release": "^1.4.0",
+    "@contentful/rich-text-html-renderer": "^16.3.0",
```

Comment:
Hmm, so this is an rtf => HTML renderer. Have you considered markdown instead? I ask because we already have a markdown renderer.

Could you explain what type of content gets rendered using this package?

Reply:
Contentful's Rich Text Format is a JSON-based document representation of rich text content, which includes text formatting options such as paragraphs, lists, blockquotes (https://www.contentful.com/developers/docs/concepts/rich-text/). Using this package to convert this RTF object to make it serializable so as to have a JSON manageable by the controller